cleverhans_tutorials/mnist_tutorial_ead.py

```diff
@@ -0,0 +1,254 @@
+"""
```

Comment:
We generally don't add new tutorials for each attack, because this leads to a lot of copy-pasted code and support / maintenance burden. Is there something conceptually unique about how to write code using this particular attack? You could also consider contributing this to the `examples` directory, where there isn't an obligation for us to provide ongoing support. (In general, remember to write to cleverhans-dev@googlegroups.com before starting work on a feature to make sure it will be accepted. New single-attack tutorials are one of the most common contributions we ask to have moved to `examples/`)